integration_tests/src/main/python/orc_write_test.py

```diff
@@ -79,6 +84,10 @@
         pytest.param([date_gen], marks=pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/139')),
         pytest.param([timestamp_gen], marks=pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/140'))]
 
+bool_gen = [pytest.param([BooleanGen(nullable=True)],
+                                 marks=pytest.mark.allow_non_gpu('ExecutedCommandExec','DataWritingCommandExec')),
+            pytest.param([BooleanGen(nullable=False)],
+                         marks=pytest.mark.allow_non_gpu('ExecutedCommandExec','DataWritingCommandExec'))]
 @pytest.mark.parametrize('orc_gens', orc_write_gens_list, ids=idfn)
```

Comment:
```suggestion

@pytest.mark.parametrize('orc_gens', orc_write_gens_list, ids=idfn)
```

Reply:
This is a nit that seemed to be missed.  Would be nice to have whitespace separating the module variables from the methods.